core/modules/taxonomy/js/taxonomy.term.js

```diff
@@ -0,0 +1,35 @@
+(function ($) {
+
+Backdrop.behaviors.Term = {
+  // Provide the vertical tab summaries.
+  attach: function (context) {
+    var $context = $(context);
+
+    // Relation settings.
+    $context.find('#edit-relations').backdropSetSummary(function(context) {
+      var vals = [];
+
+      var info = Backdrop.t('No relations.');
+      if (parents = $(context).find('select[name="parent[]"] option:selected').text()) {
+        var info = 'Parent: ' + Backdrop.checkPlain(parents);
```

Comment:
Needs to be using `Backdrop.t('Parents: @parents', { '@parents': parents })` here. (Not sure if that syntax is 100% correct, so this isn't a GitHub suggestion).

Reply:
This works for me:

```suggestion
        var info = Backdrop.t('Parent: @parents', {'@parents': Backdrop.checkPlain(parents)});
```

Reply:
The use of the `@` prefix makes it escaped already, so there's no need to keep the `checkPlain()` call.

Reply:
I ran into another issue where multiple parents weren't being handled correctly and instead all the parents were being concatenated without any separator. I pushed up an update that should fix both the translation issue and the multiple parents issue.